src/chart/draw/consolidateData/setDomain.js

```diff
@@ -0,0 +1,63 @@
+import naturalSorter from '../../../dataOps/naturalSorter';
```

Comment:
Would be good to add some comments to this code to make it easier to follow.

Reply:
This is very nice. ConsolidateData is *a lot* easier to follow now. Well done. 💯

Reply:
True!  I'll add 'em in `falsey-zeroes`.